scripts/pkg/generate-choco-package.js

```diff
@@ -6,7 +6,7 @@
 
 require('essentials');
 
-const argv = require('minimist')(process.argv.slice(2), {
+const argv = require('yargs-parser')(process.argv.slice(2), {
```

Comment:
Let's remove this change (should be covered in other PR)

Reply:
Doesn't seem to be addressed

Reply:
Seems not addressed

Reply:
I get this error if a add it back `error  'minimist' should be listed in the project's dependencies.` :(

Reply:
I think if you ensure there are no changes in this file, then CI should be fine (lint is applied (via `npm run lint:updated`) only to files which were modified in PR).

Anyway I've prepared a specific PR for this: https://github.com/serverless/serverless/pull/7273